docs/README.md

```diff
@@ -12,3 +12,75 @@ We currently have the following components available for React applications:
 -   [Score Card](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/ScoreCard.md)
 -   [Spacer](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/Spacer.md)
 -   [User Menu](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/UserMenu.md)
+
+---
+
+## Style Guide
+
+Material UI, and by extension, Brightlayer UI provides several different ways to customize a component's styles. Your specific context will determine which one is ideal. From narrowest to broadest use case, here are the options:
+
+do we want all these?? links to our packages?? links to mui?? copy mui docs vs linking to mui docs
+
```

Comment:
it would be nice to have links to each component doc style section...maybe?
[ChannelValue Styles](https://github.com/brightlayer-ui/react-component-library/blob/release/mui-v5-style-updates/docs/ChannelValue.md#classes)

Reply:
MUI link itms could be:
[One-off customization](https://mui.com/material-ui/customization/how-to-customize/#1-one-off-customization)
[Reusable component](https://mui.com/material-ui/customization/how-to-customize/#2-reusable-component)
[Global theme variation](https://mui.com/material-ui/customization/how-to-customize/#3-global-theme-overrides)

Reply:
I'm not sure I want to have a link to every style section component as I'm hoping it actually works in reverse where people go to the component style section and use that link to find this larger style override guide. The mui links have been peppered in where necessary